### PR TITLE
Clarifying variable names

### PR DESCRIPTION
**Pull request by SpookyBeverage**

### CODE REVIEW

File: apps/frontend/components/stepComponents/InformationStep.tsx [hunk 1]
```diff
@@ -25,22 +25,22 @@ export const InformationStep = ({ form, ...props }) => {
 					</div>
 				</InputSection>
 
-				<InputSection header={'File Output'}>
+				<InputSection header={'Trial\'s Extra File'}>
```
Comment (budak7273) [inline apps/frontend/components/stepComponents/InformationStep.tsx:28]:
```suggestion
				<InputSection header={"Trial's Extra File"}>
```

**Review by budak7273 — CHANGES_REQUESTED** [review]
the example experiments still need the field renames in their directions

File: example_experiments/python/genetic_algorithm.py [hunk 2]
```diff
@@ -21,14 +21,14 @@
 Example settings for a run that demonstrates this:
 
 Info:
-File Output: geneticRes.csv
+Trial's Extra File: geneticRes.csv
```
Comment (budak7273) [inline example_experiments/python/genetic_algorithm.py:24]: I thought `Trial's Extra File` was the replacement for `Result Output`?

Comment (SpookyBeverage) [reply example_experiments/python/genetic_algorithm.py]: Yup should `Trial Result`, Was just copy and pasting

**Review by budak7273 — APPROVED** [review]

**Pull request merged**